app/presenters/summary/html_sections/other_court_cases.rb

```diff
@@ -0,0 +1,36 @@
+module Summary
+  module HtmlSections
+    class OtherCourtCases < Sections::BaseSectionPresenter
+      def name
+        :other_court_cases
+      end
+
+      # rubocop:disable Metrics/AbcSize
+      def answers
+        return [Separator.not_applicable] unless previous_proceedings?
```

Comment:
Also we can get rid of this, as this is more for the PDF were we need to say something even if the question doesn't apply to the applicant, but in the CYA we just don't render anything.

Reply:
done